src/main/webapp/courseLoader.js

```diff
@@ -167,11 +133,66 @@
     chart.draw(comparisonData, google.charts.Bar.convertOptions(options));
   }
 
+  async function makeTermPerceptionChart(termDataObject) {
+    const [prevTerm1, prevTerm2] = await getPrevTermName();
```

Comment:
Ideally you'd probably want this to be flexible and adapt to any number of returned results, not just arbitrarily 2. This makes things much easier if/when you decide you want 3. Or 4. Or 5. Ect. Web dev is full of iteration so it's best to code with that in mind : )

Reply:
modified this so now it works for all numbers of comparisons. Currently I leave it hard coded for 2 previous terms, but I will eventually add a feature on to let them select how many they want compare across.